documentation/modules/con-kafka-connect-authentication.adoc

```diff
@@ -63,27 +65,41 @@ spec:
   # ...
   authentication:
     type: scram-sha-512
-    username: my-connect-user
+    username: my-connect-username <1>
     passwordSecret:
-      secretName: my-connect-user
-      password: my-connect-password-key
+      secretName: my-connect-secret-name <2>
+      password: my-connect-password-field <3>
   # ...
 ----
 
-== SASL based PLAIN authentication
+<1> `username`: The username.
+<2> `secretName`: The name of the Secret that contains the username and password.
+<3> `password`: The name of the field in the Secret under which the password is stored.
```

Comment:
Should we maybe add also an example of secret YAML with the password to make a better connection what should be set there?

Reply:
Good suggestion. I've added some example YAML (see below). The password is opaque.

```
kind: Secret
apiVersion: v1
metadata:
  name: my-connect-secret-name
  namespace: default
  selfLink: /api/v1/namespaces/default/secrets/my-connect-secret-name
  uid: 77261fa0-00e6-48d4-b4cd-52b20159d83d
  resourceVersion: '32419'
  creationTimestamp: '2020-08-28T14:40:46Z'
data:
  my-connect-password-field: QTxifR4uQ281cnQxMjU=
type: Opaque
```

Reply:
I added the following example. Please can you check it.

**Example Secret YAML for SCRAM-SHA-512 client authentication for Kafka Connect**
```
apiVersion: v1
kind: Secret
metadata:
  name: my-connect-secret-name
type: Opaque
data:
  my-connect-password-field: LFTIyFRFlMmU2N2Tm
```